TO-DO-APP/app/src/main/java/com/example/to_do_app/CreateCard.kt

```diff
@@ -1,39 +0,0 @@
-package com.example.to_do_app
-
-
-import android.content.Intent
-import androidx.appcompat.app.AppCompatActivity
-import android.os.Bundle
-import androidx.room.Room
-import kotlinx.android.synthetic.main.activity_create_card.*
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.launch
-
-
-class CreateCard : AppCompatActivity() {
-    private lateinit var database: myDatabase
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_create_card)
-        database = Room.databaseBuilder(
-            applicationContext, myDatabase::class.java, "To_Do"
-        ).build()
-        save_button.setOnClickListener {
-            if (create_title.text.toString().trim { it <= ' ' }.isNotEmpty()
-                && create_priority.text.toString().trim { it <= ' ' }.isNotEmpty()
-            ) {
-                var title = create_title.getText().toString()
-                var priority = create_priority.getText().toString()
-                DataObject.setData(title, priority)
-                GlobalScope.launch {
```

Comment:
Why use GlobalScope this can be done with viewModel Scope or move this to background thread by using Dispatcher.IO